website/blog/2024-05-21-quiz/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
example file is okay. however, put this file to `components_example` folder in `docs` folder, don't put it in `blog` folder because it will display in our main site for end user

<img width="1223" alt="Screenshot 2024-05-29 at 07 47 06" src="https://github.com/intelligent-investors/intelligent-investors/assets/1780281/41e65ed1-42d3-4ad7-9b16-b9a0aa64ae91">